src/repository.ts

```diff
@@ -186,6 +189,13 @@ abstract class Repository extends AsyncOptionalCreatable {
     return found;
   }
 
+  private getRegistryForAuth(): string {
+    const registry: string = this.env.getString('NPM_REGISTRY');
```

Comment:
you don't need to specify `string` here, it should be implied from `getString`

Reply:
Will remove